Makefile

```diff
@@ -219,7 +219,7 @@ manifests.crds: controller-gen ## Generate WebhookConfiguration and CustomResour
 
 .PHONY: manifests.rbac ## Generate ClusterRole objects.
 manifests.rbac: controller-gen
-	$(CONTROLLER_GEN) rbac:roleName=kong-ingress paths="./internal/controllers/configuration/"
+	$(CONTROLLER_GEN) rbac:roleName=kong-ingress paths="./internal/controllers/configuration/" paths="./controllers/license/"
```

Comment:
I wasn't aware of the fact that you can specify `paths` multiple times 🤔 

https://github.com/Kong/gateway-operator/blob/3711c3ec5d73c597b7c1fbb62aa2c87a8bec26b7/Makefile#L299-L307

Do we want to go this route or would we prefer splitting with `;` and using `paths` once?

Reply:
I tried with separator and `,`, ` ` or `;` doesn't work. It was achieved via tries and errors

Reply:
Weird 🤔 It worked like this (split with `;`) for a long time in KGO.

Reply:
So maybe I was splitting wrongly
```
paths="./internal/controllers/configuration/; ./controllers/license/"
```
something like the above didn't work